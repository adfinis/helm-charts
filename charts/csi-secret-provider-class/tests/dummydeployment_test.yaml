suite: deployment tests
templates:
  - deployment.yaml
tests:
  - it: should match snapshot
    values:
      - ../ci/default-values.yaml
    set:
      dummyDeployment.enabled: true
    asserts:
      - hasDocuments:
          count: 1
      - matchSnapshot: {}
