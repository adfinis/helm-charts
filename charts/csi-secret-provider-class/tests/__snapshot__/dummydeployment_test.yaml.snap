should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/app: csi-secret-provider-class
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: 0.3.0
        helm.sh/chart: csi-secret-provider-class-0.3.1
      name: release-name-csi-secret-provider-class
    spec:
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: csi-secret-provider-class
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: csi-secret-provider-class
        spec:
          containers:
          - image: k8s.gcr.io/pause:3.4.1
            imagePullPolicy: IfNotPresent
            name: dummy
            volumeMounts:
            - mountPath: /vault
              name: csi-secret-store-volume
              readOnly: true
          volumes:
          - csi:
              driver: secrets-store.csi.k8s.io
              readOnly: true
              volumeAttributes:
                secretProviderClass: release-name-csi-secret-provider-class
            name: csi-secret-store-volume
